src/chart/heatmap/HeatmapView.ts

```diff
@@ -205,6 +206,8 @@ class HeatmapView extends ChartView {
         let emphasisStyle = seriesModel.getModel(['emphasis', 'itemStyle']).getItemStyle();
         let blurStyle = seriesModel.getModel(['blur', 'itemStyle']).getItemStyle();
         let selectStyle = seriesModel.getModel(['select', 'itemStyle']).getItemStyle();
+        const itemStyle = seriesModel.getModel(['itemStyle']);
```

Comment:
No need to get a temporary model here. Just `seriesModel.get(['itemStyle', borderRadius']);`

Reply:
Also `borderRadius` should support to be configured per data item. Like `emphasisStyle`, we can override the `borderRadius` in the `data.hasItemOption` branch below.

Reply:
copy that